Makefile

```diff
@@ -296,17 +345,3 @@ bump:
 
 e2e-upload:
 		./tools/code-sign.sh -- $(MAKE) -C E2ETests
-
-# Helpers
-
-ECHO_TITLE=./tools/utils/echo_color.sh --title
-ECHO_ERROR=./tools/utils/echo_color.sh --err
-ECHO_WARNING=./tools/utils/echo_color.sh --warn
-ECHO_SUCCESS=./tools/utils/echo_color.sh --succ
-
-define require_param
-    if [ -z "$${$(1)}" ]; then \
-        $(ECHO_ERROR) "Error:" "$(1) parameter is required but not provided."; \
-        exit 1; \
-    fi
-endef
```

Comment:
💡 Moved to `tools/utils/common.mk` as now being shared with `Makefiles` used for smoke testing.